pkg/flink/internal/controller/input_controller.go

```diff
@@ -169,14 +173,10 @@ func (c *InputController) Prompt() prompt.IPrompt {
 		prompt.OptionSetStatementTerminator(func(lastKeyStroke prompt.Key, buffer *prompt.Buffer) bool {
 			text := buffer.Text()
 			text = strings.TrimSpace(text)
-			// We add exit here because we also want to exit without the need of adding semicolon, which is the default flow for all statements
-			if text == "exit" {
-				return true
-			}
-			if text == "" || !strings.HasSuffix(text, ";") {
+			if text == "" {
 				return false
 			}
-			return true
+			return text == "exit" || strings.HasSuffix(text, ";") || lastKeyStroke == prompt.AltEnter
```

Comment:
This would let us delete the `if` above... just a suggestion!
```suggestion
			return text == "exit" || strings.HasSuffix(text, ";") || text != "" && lastKeyStroke == prompt.AltEnter
```

Reply:
In boolean logic, and always comes before or regardless of language! Happy to leave things as is for extra clarity though

Reply:
Sounds good, I think having the extra if makes it tiny bit more easier to understand